solar_system.rb

```diff
@@ -0,0 +1,111 @@
+# class for solarsystem that can contain mulitple planets
+class SolarSystem
+
+  attr_accessor :planets
+
+  def initialize(planet_names)
+    @planets = planet_names
+  end
+
+  def add_planet(new_planet)
+    @planets << new_planet
+  end
+
+  def planets_listed
+    n = 1
+    listed_planets = ""
+    @planets.each do |planet|
+      listed_planets += "#{n}.\t #{planet.name}\n"
+      n += 1
+    end
+    return listed_planets
+  end
+
+  def return_specific_planet(planet_num)
+    return @planets[planet_num -1]
+  end
+end
+
+# class for planet that sets up a new planet
+class Planet
+  attr_reader :name, :terraform_state, :temperature, :atmosphere, :location
+
+  def initialize(name, terraform_state, temperature, atmosphere, location)
+    @name = name
+    @terraform_state = terraform_state
+    @temperature = temperature
+    @atmosphere = atmosphere
+    @location = location
+  end
+
+  # Array style of returning planet info
+  def return_planet
+    terrraform_info = ""
+    if @terraform_state
+      terrraform_info = "Additionally, it has been terraformed."
+    else
+      terrraform_info = "Additionally, it has not been terraformed."
+    end
+    planet_info = "#{@name} is a planet located in the #{@location} galaxy. It is a #{@temperature} planet, whose atmosphere is #{@atmosphere} to humans. #{terrraform_info}"
+    return planet_info
+  end
+end
+# Hard coded data that pre-populates solar system
+
+venus = Planet.new("Venus", false, "hot", "unbreathable", "Milky Way")
+
+jupiter = Planet.new("Mercury", false, "hot", "unbreathable", "Milk Way")
+
+pluto = Planet.new("Pluto", false, "hot", "unbreathable", "Milky Way")
+
+planet_list = [
+  venus,
+  jupiter,
+  pluto
+  ]
+
+# initializes solar system
+my_solar_system = SolarSystem.new(planet_list)
+
+puts "Welcome to the Solar System 2000, where you can learn information about the planets in the database or you can add in your own planets!"
```

Comment:
It would be good to put this in some kind of loop that would let us to look at a number of planets.